Source/WebKitLegacy/mac/WebCoreSupport/WebEditorClient.mm

```diff
@@ -356,10 +356,6 @@ static void updateFontPanel(WebView *webView)
     }
 
 #if !PLATFORM(IOS_FAMILY)
-    // FIXME: This quirk is needed due to <rdar://problem/5009625> - We can phase it out once Aperture can adopt the new behavior on their end
-    if (!WebKitLinkedOnOrAfter(WEBKIT_FIRST_VERSION_WITHOUT_APERTURE_QUIRK) && [[[NSBundle mainBundle] bundleIdentifier] isEqualToString:@"com.apple.Aperture"])
```

Comment:
Please delete the `#define WEBKIT_FIRST_VERSION_WITHOUT_APERTURE_QUIRK` in `Source/WebKitLegacy/Misc/WebKitVersionChecks.h`, too.